code/game/objects/items/radio/headset.dm

```diff
@@ -202,11 +200,16 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	return ..()
 
 
-/obj/item/radio/headset/mainship/proc/toggle_squadhud(mob/living/carbon/human/user)
+/obj/item/radio/headset/mainship/proc/toggle_squadhud()
+	if(!ishuman(loc))
```

Comment:
Not a huge fan of this change.
Instead of toggling it off and on again each time it's equipped, it would be toggled off on unequip.